xwiki-platform-core/xwiki-platform-bridge/src/main/java/org/xwiki/bridge/event/UserAuthenticatedEvent.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.bridge.event;
+
+import org.xwiki.model.reference.DocumentReference;
+
+/**
+ * An event triggered when a user is authenticated.
+ * <p>
+ * The event also send the following parameters:
+ * </p>
+ * <ul>
+ * <li>source: the current {com.xpn.xwiki.user.api.XWikiUser} instance which has been authenticated.</li>
+ * <li>data: the current {com.xpn.xwiki.XWikiContext} instance</li>
+ * </ul>
+ * 
+ * @version $Id$
+ * @since 2.7RC1
+ */
+public class UserAuthenticatedEvent extends AbstractDocumentEvent
+{
+    /**
+     * The version identifier for this Serializable class. Increment only if the <i>serialized</i> form of the class
+     * changes.
+     */
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Constructor initializing the event filter with a {@link org.xwiki.observation.event.filter.FixedNameEventFilter},
+     * meaning that this event will match only delete events affecting the same user.
+     *
+     * @param userReference the reference of the user to match
+     */
+    public UserAuthenticatedEvent(DocumentReference userReference)
```

Comment:
Don't we want to work with UserReference instead ? I'm surprised @vmassol did not commented about that :)

Reply:
> I'm surprised @vmassol did not commented about that :)

I did actually ;)